Source/Core/Common/CMakeLists.txt

```diff
@@ -205,13 +205,18 @@ if (APPLE)
 elseif(WIN32)
   target_link_libraries(common
   PRIVATE
-    kernel32.lib
-    shlwapi.lib
-    winmm.lib
+    kernel32
+    shlwapi
+    winmm
+    SetupAPI
   )
   if (_M_X86_64)
-    target_link_libraries(common PRIVATE opengl32.lib)
+    target_link_libraries(common PRIVATE opengl32)
   endif()
+	if(MINGW)
+		target_compile_definitions(common PRIVATE _WIN32_WINNT=0x0602)
+		target_compile_options(common PRIVATE -mxsave)
+	endif()
```

Comment:
Spaces should be used instead of tabs as this file doesn't seem to have any.

Reply:
done